### ISSUE TITLE
configurable user agent

### ISSUE THREAD

**Issue by balcy** [ubports/webbrowser-app#9]
for some use cases it helps to be able to use different user agents, e.g Desktop Chrome 
(Windows), to unblock certain site features that do technically work.
E.g. the website of my VOIP provider will not allow browser calls for the UBPorts browser, but state "sorry currently only Chrome and Firefox are supported"
Testing with a different user agent (app that uses Ubuntu WebView) it worked fine.

**Comment by delijati** [comment]
done with #23

**Comment by balcy** [comment]
thanks : ]

**Comment by mihaelmilea** [comment]
Where can I see this? Only in Xenial?

**Comment by UniversalSuperBox** [comment]
webbrowser-app has been replaced with [morph-browser](https://github.com/ubports/morph-browser). Please file any new issue reports there.
